airbyte-integrations/bases/base-normalization/normalization/destination_type.py

```diff
@@ -26,7 +26,11 @@
 
 
 class DestinationType(Enum):
-    bigquery = "bigquery"
-    postgres = "postgres"
-    redshift = "redshift"
-    snowflake = "snowflake"
+    BIGQUERY = "bigquery"
```

Comment:
What does the casing of the value change things make things clearer?

I just uppercase the name of the enum because:

> Note
> Case of Enum Members
> Because Enums are used to represent constants we recommend using UPPER_CASE names for enum members, and will be using that style in our examples.
https://docs.python.org/3/library/enum.html